foundations/installations/installing_node.md

```diff
@@ -66,7 +99,7 @@ nvm install node
 This will install the most recent version of node, and you’ll see a lot of output in the terminal. If everything worked, you should see something similar to this somewhere in the lines of output:
 
 ~~~bash
-Downloading and installing node v12.6.0...
+Downloading and installing node v14.15.4...
```

Comment:
Have we tested this new version with the content in the curriculum?

Reply:
The command mentioned above installs this version of node, so I edited it to the Semantic versioning.

Reply:
If that is the case, you should also edit the lines below this line [here](https://github.com/TheOdinProject/curriculum/pull/22117/files#diff-d21436e26ed270deb4cdcd7b13f4e478f79095b3335604f55d65dc5c8dc787b0R114) to match this version. 

Maybe to keep it more flexible, replace the `.4` with `.xx` and explain what that means so we don't have to update this every month.